definitions/features/katello.rb

```diff
@@ -59,6 +61,16 @@ def config_files_exclude_for_online
     ]
   end
 
+  def repos_rpm(version, os_release)
+    "https://fedorapeople.org/groups/katello/releases/yum/#{version}/katello/el#{os_release}"\
```

Comment:
If you can access `os_release` directly from `ForemanMaintain::Utils::Facter.os_major_release` so would it be possible to remove  method argument `os_release`?

Reply:
The os_release is being used many times to generate repo links. So instead of calling `ForemanMaintain::Utils::Facter.os_major_release` separately and repeatedly in each feature I thought I should collect it once and pass it ?

Reply:
Then in my opinion it would be better to move this method inside `instance` feature than `upstream_repositories`. What do you think?